src/deye_events.py

```diff
@@ -68,3 +98,32 @@ def get_id(self):
     @abstractmethod
     def process(self, events: list[DeyeEvent]):
         pass
+
+
+def events_to_str(events: list[DeyeEvent] | set[DeyeEvent]) -> str:
+    return ", ".join([str(e) for e in events])
+
+
+def compare_event_list(events_a: list[DeyeEvent], events_b: list[DeyeEvent], check_status: bool = False) -> bool:
```

Comment:
Please define methods within classes. For example, move this method to `DeyeEventsComparator` class

Reply:
In order to not just encapsulate the single compare  function in a class, I propose to define a list sub-class "DeyeEvents(list)" which extends the list class with all the required methods like compare, get_status_events, `__str__`.
This would impact all current usage of `list[DeyeEvent]`  to use `DeyeEvents` instead, but looks cleaner to me.

Reply:
Alright, I suggest to use more self-explanatory name `DeyeEventList`

Reply:
implemented new class with required methods